sdk/communication/Azure.Communication.Chat/api/Azure.Communication.Chat.netstandard2.0.cs

```diff
@@ -142,19 +151,15 @@ public partial class ChatThreadProperties
         public string Id { get { throw null; } }
         public string Topic { get { throw null; } }
     }
-    public partial class CommunicationError
-    {
-        internal CommunicationError() { }
-        public string Code { get { throw null; } }
-        public System.Collections.Generic.IReadOnlyList<Azure.Communication.Chat.CommunicationError> Details { get { throw null; } }
-        public Azure.Communication.Chat.CommunicationError InnerError { get { throw null; } }
-        public string Message { get { throw null; } }
-        public string Target { get { throw null; } }
-    }
     public partial class CreateChatThreadResult
     {
         internal CreateChatThreadResult() { }
         public Azure.Communication.Chat.ChatThreadProperties ChatThreadProperties { get { throw null; } }
```

Comment:
Name of the field : ChatThread
public Azure.Communication.Chat.ChatThreadProperties ChatThread { get { throw null; } }

Reply:
changed